src/main/kotlin/org/jitsi/nlj/RtpReceiverConfig.kt

```diff
@@ -0,0 +1,23 @@
+package org.jitsi.nlj
+
+import org.jitsi.config.newConfigAttributes
+import org.jitsi.utils.config.SimpleProperty
+
+class RtpReceiverConfig {
+    class Config {
+        companion object {
+
+            class RtpReceiverQueueSizeProperty : SimpleProperty<Int>(
+                    newConfigAttributes {
+                        readOnce()
+                        name("jmt.transceiver.recv.q-size")
+                    }
+            )
+            private val rtpReceiverQueueSizeProperty =
+                    RtpReceiverQueueSizeProperty()
+
+            @JvmStatic
+            fun rtpReceiverQueueSizeProperty(): Int = rtpReceiverQueueSizeProperty.value
```

Comment:
We usually name these without "Property" at the end

Reply:
Possibly without "rtpReceiver" too, since it's in the "RtpReceiverConfig" class already

Reply:
does that apply to everything? class names, function names and field names?

Reply:
No, just the public function (so we can do `Config.queueSize()` instead of `Config.rtpReceiverQueueSizeProperty()`)